jrnl/plugins/yaml_exporter.py

```diff
@@ -80,9 +77,30 @@ def export_entry(cls, entry, to_multifile=True):
         dayone_attributes = ""
         if hasattr(entry, "uuid"):
```

Comment:
Does every Day One entry have a uuid? If so, can we indent all of the following conditionals so that they're nested inside of this conditional? That way, we don't check the conditions for all entries if we know they aren't related to Day One.

Reply:
It should, but I'm leery of assuming that will always be the case.

It may make sense to indent lines 88 through 99 in any case.

Reply:
I've intended the second block of lines in 31a040878c6bc2b302c0e7709e305e3955701b6f